multinet/errors.py

```diff
@@ -242,3 +242,15 @@ def __init__(self, message: str = ""):
     def flask_response(self) -> FlaskTuple:
         """Generate a 400 error."""
         return (self.message, "400 Error during AQL Execution")
+
+
+class NonExistantUploadDocument(ServerError):
```

Comment:
I believe there's a `NotFound` base class you can use here. Not including a code suggestion because the rest of the class would need to change as well (it would become simpler, IIUC).